tests/test_geodesic.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+import os.path as op
+
+import matplotlib.pyplot as plt
+import plenoptic as po
+import pytest
+import torch
+
+from test_plenoptic import DATA_DIR, DEVICE, DTYPE
+
+
+class TestGeodesic(object):
+
+    def test_brownian_bridge(self):
+        """Note: this is a stochastic test that will be true with high proba"""
```

Comment:
why not set seed?

Reply:
could do a handful of seeds you know will pass

Reply:
(if there's some reason you don't want to do just one)